### ISSUE TITLE
Question about post-export extension in regard to renew function

### ISSUE THREAD

**Issue by muemi03** [Layer7-Community/graphman-client#73]
Hi Team.
Just a question:
I am using the post-export extension to do specific manipulation to an exported bundle.
Now I am looking at the renew function.
Obviously the "post-export" extension isn't called for a renew.
Wouldn't that make sense because the renew is exporting entities as well.
Or providing a "post-renew" extension ?

The principle I  have in mind is having an extension which gets called for any kind of "getting entities from  gateway?

Even though this understanding would impact the diff command as well, but I am looking at renew only currently.

Thanks
...Michael

**Comment by graju256** [comment]
@muemi03 I agree. We will make sure to have this post-export extension get called for the relevant places.

Thanks.
